### PR TITLE
Add ovirt guest agent info for Atomic hosts

### PR DESCRIPTION
**Pull request by lambfrier**
No information is easily available for getting the ovirt-guest-agent running on Atomic (or any other container distributions) - having these pointer commands may help others save a couple of hours.

*Note*: at the time of writing, the guest agent doesn't start since the OCI spec is old.  I'm not sure how to file a bug for this.  Basically, it seems the Docker Hub image at https://hub.docker.com/r/ovirtguestagent/centos7-atomic/ isn't up-to-date with the repo. Specifically, the config.json file isn't in-line with https://github.com/projectatomic/atomic-system-containers/blame/master/ovirt-guest-agent-centos/config.json.template

Changes proposed in this pull request:
- add small section for ovirt guest agent on atomic hosts

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @lambfrier

### CODE REVIEW

File: source/documentation/vmm-guide/chap-Installing_Linux_Virtual_Machines.html.md [hunk 1]
```diff
@@ -308,6 +308,22 @@ The ovirt guest agents and drivers are installed on Enterprise Linux virtual mac
...
+
+   The commands for a Centos7 Atomic host are essentially:
+   
+         # atomic pull --storage=ostree ovirtguestagent/centos7-atomic
+         # atomic install --system --system-package=no --name=ovirt-guest-agent ovirtguestagent/centos7-atomic
+         # systemctl status ovirt-guest-agent
+         # systemctl start ovirt-guest-agent
+
+   Note: there is [official documentation and separate image registry](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/managing_containers/running_system_containers#using_the_ovirt_guest_agent_system_container_image_for_red_hat_virtualization) for those with a RHEV subscription.
+
```
Comment (sandrobonazzola) [inline source/documentation/vmm-guide/chap-Installing_Linux_Virtual_Machines.html.md]: Please use RHV instead of RHEV.

Comment (lambfrier) [reply source/documentation/vmm-guide/chap-Installing_Linux_Virtual_Machines.html.md]: Right - sorry.  Patch updated.

**Comment by lambfrier** [comment]
Thanks!
Any advice on how to get the ovirt guest agent image on Docker Hub updated?..

**Comment by sandrobonazzola** [comment]
@lambfrier please open a bug on bugzilla.redhat.com on ovirt-guest-agent product under oVirt classification.

**Pull request merged**